pack/tcu/tcu.json

```diff
@@ -0,0 +1,277 @@
+		"pack_code": "tcu",
+		"position": 27,
+		"quantity": 2,
+		"subname": "Free from the Past",
+		"text": "You get +1 [intellect].]\n[reaction] When the game begins, if Death • XIII is in your opening hand: Put it into play.",
+		"slot": "Tarot",
+		"traits": "Tarot.",
+		"type_code": "asset",
+		"xp": 1
+	},
+	{
+		"code": "05031",
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "rogue",
+		"flavor": "Open your eyes, and listen to your shadow.",
+		"illustrator": "Chris Peuler",
+		"name": "The Moon • XVIII",
+		"pack_code": "tcu",
+		"position": 31,
+		"quantity": 2,
+		"subname": "Message from Your Inner Self",
+		"text": "You get +1 [agility].\n[reaction] When the game begins, if The Moon • XVIII is in your opening hand: Put it into play.",
+		"slot": "Tarot",
+		"traits": "Tarot.",
+		"type_code": "asset",
+		"xp": 1
+	},
+	{
+		"code": "05036",
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "survivor",
+		"illustrator": "Jeff Lee Johnson",
+		"name": "Track Shoes",
+		"pack_code": "tcu",
+		"position": 36,
+		"quantity": 2,
+		"skill_agility": 1,
+		"text": "Limit 1 [[Footwear]] per investigator.\nYou get +1 [agility].\n[reaction] After you move, but before enemies at your new location engage you, exhaust Track Shoes: Test [agility] (3). If you succeed, move to a connecting location.",
+		"traits": "Item. Clothing. Footwear.",
+		"type_code": "asset",
+		"xp": 0
+	},
+	{
+		"code": "05038",
+		"deck_limit": 2,
+		"faction_code": "survivor",
+		"illustrator": "Anna Steinbauer",
+		"name": "Able Bodied",
+		"pack_code": "tcu",
+		"position": 38,
+		"quantity": 2,
+		"skill_agility": 1,
+		"skill_combat": 1,
+		"text": "While you control 2 or fewer [[Item]] assets, Able Bodied gains [combat] [agility] (while you control 1 or fewer [[Item]] assets, Able Bodied gains [combat] [combat] [agility] [agility], instead).",
+		"traits": "Innate.",
+		"type_code": "skill",
+		"xp": 0
+	},
+	{
+		"code": "05039",
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "survivor",
+		"flavor": "It is time to rise up again.",
+		"illustrator": "Chris Peuler",
+		"name": "Five of Pentacles",
+		"pack_code": "tcu",
+		"position": 39,
+		"quantity": 2,
+		"subname": "From the Brink",
+		"text": "You get +1 health and +1 sanity.\n[reaction] When the game begins, if Five of Pentacles is in your opening hand: Put it into play.",
+		"slot": "Tarot",
+		"traits": "Tarot.",
+		"type_code": "asset",
+		"xp": 1
+	},
+	{
+		"code": "05040",
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "neutral",
+		"flavor": "It is time to rise up again.",
+		"illustrator": "Robert Lackey",
+		"name": "Ace of Rods",
+		"pack_code": "tcu",
+		"position": 40,
+		"quantity": 2,
+		"subname": "The Fateful Step",
+		"text": "[fast] During your turn, remove Ace of Rods from the game: You may take an additional action this turn, during which you get +2 shroud to each of your skills.\n[reaction] When the game begins, if Ace of Rods is in your opening hand: Put it into play.",
```

Comment:
You've added the word 'shroud' in the middle here, after '+2'!

Reply:
lol, where did that come from